### ISSUE TITLE
Confirm successfully received messages?

### ISSUE THREAD

**Issue by dertester** [maniacbug/RF24Network#16]
Hi,

is there a function included to find out if a message is recieved?

From the example:
bool ok = network.write(header,&payload,sizeof(payload));

i figured out, that it only shows me true if the direct parent got the message.

Problem if i send from 031 to 02 (031->01->00->02) it will only confirm that 01 got the message even if the message never arrived at 00 or 02.

thx for any help:)

**Comment by heinemml** [comment]
I guess the point here is: the confirmation you get is part of the Chips internal protocol. So it's more or less included for free. A confirmation on top of RF24Network is not build in. You would need to send back a confirmation yourself.

**Comment by hayden-t** [comment]
that is correct, i had this desire, and i just made the destination send back a message to the source to indicatate it recieved it, and made the source keep sending at interval until it got notification
